launch-game.sh

```diff
@@ -25,7 +25,16 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		LOGS="~/Library/Application Support/OpenRA/Logs/"
+	else
+		LOGS="${XDG_CONFIG_HOME:-${HOME}/.config}/openra/Logs"
+		if [[ ! -d "${LOGS}" && -d "${HOME}/.openra/Logs" ]]; then
+			LOGS="${HOME}/.openra/Logs"
+		fi
+	fi
```

Comment:
Unfortunately, I have another correction. I didn't test this before writing the previous comment, and plain sh is quite restrictive. This is tested now, though:

```suggestion
        if [ "$OSTYPE" = "darwin"* ]; then
                LOGS="~/Library/Application Support/OpenRA/Logs/"
        else
                LOGS="${XDG_CONFIG_HOME:-${HOME}/.config}/openra/Logs"
                if [ ! -d "${LOGS}" ] && [ -d "${HOME}/.openra/Logs" ]; then
                        LOGS="${HOME}/.openra/Logs"
                fi
        fi
```